etc/profile-a-l/lobster.profile

```diff
@@ -0,0 +1,40 @@
+# Firejail profile for lobster
+# Description: Shell script to watch Movies/Webseries/Shows from the terminal
+# This file is overwritten after every install/update
+quiet
+# Persistent local customizations
+include lobster.local
+# Persistent global definitions
```

Comment:
Now that this is a redirect profile, it's important to `avoid including globals.local twice`. Here's what we use in other profiles for that purpose:
```
# added by included profile
#include globals.local
```